stingray/lightcurve.py

```diff
@@ -1059,7 +1059,7 @@ def plot(self, witherrors=False, labels=None, axis=None, title=None,
         if title is not None:
             plt.title(title)
 
-        if save:
+        if save is True:
```

Comment:
Just `if save` is sufficient. If one writes `save='mickeymouse'` when the docs say that `save` is `bool`... well... whatever happens, they had it coming :)

Reply:
Done. :+1: